recipes/fiftyone-brain/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "fiftyone-brain" %}
+{% set version = "0.16.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiftyone-brain-{{ version }}.tar.gz
+  sha256: a3f52bf5b7be1516f7b5202b434d8e15cf60e570b1b9d13652a0ad9610864b67
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3
```

The metadata has 3.8

```suggestion
    - python >=3.8
```